src/app/modules/gb-analysis-module/cross-table-components/gb-droppable-zone/gb-droppable-zone.component.ts

```diff
@@ -7,13 +7,13 @@
  */
 
 import {Component, Input, OnInit} from '@angular/core';
-import {Constraint} from '../../../models/constraint-models/constraint';
-import {CrossTableService} from '../../../services/cross-table.service';
-import {TreeNodeService} from '../../../services/tree-node.service';
-import {ConstraintService} from '../../../services/constraint.service';
-import {DropMode} from '../../../models/drop-mode';
-import {MessageHelper} from '../../../utilities/message-helper';
-import {AxisType} from '../../../models/table-models/axis-type';
+import {Constraint} from '../../../../models/constraint-models/constraint';
+import {CrossTableService} from '../../../../services/cross-table.service';
+import {TreeNodeService} from '../../../../services/tree-node.service';
+import {ConstraintService} from '../../../../services/constraint.service';
+import {DropMode} from '../../../../models/drop-mode';
```

Comment:
A bit less related to your PR, but I wonder what the [`DropMode`](https://github.com/thehyve/glowing-bear/blob/b55b2b13fcb48d7d3af3685735b5ba7232088549/src/app/models/drop-mode.ts) enum does, it looks like only the `DropMode.TreeNode` is used. Should it be updated to have `Cohort` and `Variable` modes instead of some old ones? Or removed?